MapboxMobileEvents/MMENSURLSessionWrapper.m

```diff
@@ -46,12 +46,36 @@ - (void)processRequest:(NSURLRequest *)request completionHandler:(void (^)(NSDat
 #pragma mark NSURLSessionDelegate
 
 - (void)URLSession:(NSURLSession *)session didReceiveChallenge:(NSURLAuthenticationChallenge *)challenge completionHandler:(void (^)(NSURLSessionAuthChallengeDisposition, NSURLCredential * _Nullable))completionHandler {
-    // Call into TrustKit here to do pinning validation
-    if (![self.trustKit.pinningValidator handleChallenge:challenge completionHandler:completionHandler]) {
-        // TrustKit did not handle this challenge: perhaps it was not for server trust
-        // or the domain was not pinned. Fall back to the default behavior
-        completionHandler(NSURLSessionAuthChallengePerformDefaultHandling, nil);
-    }
+    __block void (^completion)(NSURLSessionAuthChallengeDisposition, NSURLCredential * _Nullable) = nil;
+    
+    completion = ^(NSURLSessionAuthChallengeDisposition disposition, NSURLCredential * _Nullable credential){
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
For comments below - the completion block is dispatched to the main queue (on the main thread), which is what we want.